interface/forms/procedure_order/common.php

```diff
@@ -0,0 +1,1287 @@
+    }
+    let printer = 'file';
+    let acctid = '<?php echo $gbl_client_acct ?>';
+    let order = f.id.value;
+    let patient = "<?php echo $patient['lname'] . ', ' . $patient['fname'] . ' ' . $patient['mname'] ?>";
+    let pid = "<?php echo $patient['pid']  ?>";
+    let url = top.webroot_url + "/interface/procedure_tools/libs/labs_ajax.php";
+    // this escapes above
+    let uri = encodeURI("?action=print_labels&count=" + count + "&order=" + order + "&pid=" + pid +
+        "&acctid=" + acctid + "&patient=" + patient + "&specimen=" + tarray) +
+        "&csrf_token_form=" + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>;
+
+    // retrieve the labels
+    dlgopen(url + uri, 'pdf', 'modal-md', 750, '');
+}
+
+function doWait(e){
+    $(".wait").removeClass('d-none');
+    return true;
+}
+</script>
+<style>
+    @media only screen and (max-width: 768px) {
+        [class*="col-"] {
+            width: 100%;
+            text-align: left !important;
+        }
+    }
+
+    .qoe-table {
+        margin-bottom: 0px;
+    }
+
+    .proc-table {
+        margin-bottom: 5px;
+    }
+
+    .proc-table .itemDelete {
+        width: 25px;
+        color: var(--danger);
+        cursor: pointer;
+    }
+
+    .proc-table .itemTransport {
+        width: 45px;
+        margin: 5px 2px;
+        padding: 2px 2px;
+        cursor: hand;
+    }
+
+    .proc-table .procedure-div {
+        min-width: 40%;
+    }
+
+    .proc-table .diagnosis-div {
+        min-width: 20%;
+    }
+
+    .c-hand {
+        cursor: pointer;
+    }
+</style>
+</head>
+<?php
+$name = $enrow['fname'] . ' ';
+$name .= (!empty($enrow['mname'])) ? $enrow['mname'] . ' ' . $enrow['lname'] : $enrow['lname'];
+$date = xl('on') . ' ' . oeFormatShortDate(substr($enrow['date'], 0, 10));
+$title = array(xl('Order for'), $name, $date);
+?>
+<body class="body_top" onsubmit="doWait(event)">
+    <div class="container">
+        <div class="page-header">
+            <h2><?php echo implode(" ", $title); ?></h2>
+        </div>
+        <div class="col-md-12">
+            <form class="form form-horizontal" method="post" action="" onsubmit="return validate(this,event)">
+                <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                <input type='hidden' name='id' value='<?php echo $formid ?>' />
+                <fieldset class="row">
+                    <legend data-toggle="collapse" data-target="#orderOptions">
+                        <i class="fa fa-plus"></i>
+                        <?php echo xlt('Select Options for Current Order Id ') . (text($formid) ? text($formid) : 'New Order') ?>
+                    </legend>
+                    <div class="col-md-12 collapse show" id="orderOptions">
+                        <div class="form-group form-row">
+                            <label for="provider_id" class="col-form-label col-md-2"><?php xl('Ordering Provider', 'e'); ?></label>
```

Comment:
the e parameter in xl has been deprecated since impossible to escape them (in these cases, need to add echo, remove the e, and then change to xlt/xla):
```
 <label for="provider_id" class="col-form-label col-md-2"><?php echo xlt('Ordering Provider'); ?></label>
```